lib/phoenix_html/form.ex

```diff
@@ -1300,6 +1301,13 @@ defmodule Phoenix.HTML.Form do
   defp date_value({year, month, day}), do: %{year: year, month: month, day: day}
 
   defp date_value(nil), do: %{year: nil, month: nil, day: nil}
+
+  defp date_value("" <> string) do
```

Comment:
`is_binary()`?

Reply:
Will change, this is just a shortcut i've used